src/test/java/io/cdap/Actions.java

```diff
@@ -0,0 +1,31 @@
+package io.cdap;
+
+import java.util.List;
+
+public class Actions {
```

Comment:
Where is this being used?

Reply:
used in the SpecJsonTestBean which is ultimately used to unmarshal the json file in the SpecJsonCreatorTest